projects/ui/src/Components/Apis/ApisTab/ApiSummaryCards/ApiSummaryListCard.tsx

```diff
@@ -32,7 +35,7 @@ export function ApiSummaryListCard({ api }: { api: APIProduct }) {
                 )}
               </div> */}
               {/* <div className="description">{api.description}</div> */}
-              {api.apiVersions.map((apiVersion) => {
+              {/* {api.apiVersions.map((apiVersion) => {
```

Comment:
Why not just completely remove?  

Perhaps it would help to leave comments for these if they are just "coming back soon"?

Reply:
I'll give these cards an update in the next PR and remove these commented sections - the data type changed for Api Products so some of the data being shown is changing.